tools/doc/html.js

```diff
@@ -152,6 +158,16 @@ function parseLists(input) {
   return output;
 }
 
+function parseYAML(text) {
+  const meta = common.extractAndParseYAML(text);
+  let html = '<div class="api_metadata">';
+
+  Object.keys(meta).forEach(key => {
+    html += '<span>' + key + ': ' + meta[key] + '</span>';
+  });
```

Comment:
@benjamingr, that is a nifty line of code :+1:

I changed it to a basic if statement though. Allows for future, per key rendering I suppose.